EFCore/MAUI/Readme.md

```diff
@@ -0,0 +1,856 @@
+  
+  > **NOTE** 
+  >
+  > If you have a pre-release version of our components, for example, provided with the hotfix, you also have a pre-release version of NuGet packages. These packages will not be restored automatically and you need to update them manually as described in the [Updating Packages](https://docs.devexpress.com/GeneralInformation/118420/Installation/Install-DevExpress-Controls-Using-NuGet-Packages/Updating-Packages) article using the [Include prerelease](https://docs.microsoft.com/en-us/nuget/create-packages/prerelease-packages#installing-and-updating-pre-release-packages) option.
+
+## Table of Contents
+
+[**Build and Test a Basic Web API Service App (Backend)**](#build-and-test-a-basic-web-api-service-app-backend)
+
+- [Create a Web API project using the XAF Solution Wizard](#create-a-web-api-project-using-the-xaf-solution-wizard)
+- [Declare a data model](#declare-a-data-model)
+- [Set up a development database connection](#set-up-a-development-database-connection)
+- [Generate initial data](#generate-initial-data)
+- [Use Swagger UI to test the Web API service](#use-swagger-ui-to-test-the-web-api-service)
+
+[**Build and test a basic .NET MAUI App (Frontend)**](#build-and-test-a-basic-net-maui-app-frontend)
+
+- [Create a .NET MAUI project](#create-a-net-maui-project)
+- [Modify platform specific settings](#modify-the-platform-specific-settings)
+- [Define the application's data model](#define-the-applications-data-model)
+- [Implement and test first views: "Login" and "Item List"](#implement-and-test-first-views-login-and-item-list)
+
+[**Extend backend and frontend apps**](#extend-backend-and-frontend-apps)
+
+- [Add an "Item Details" View](#add-an-item-details-view)
+- [Add a "New Item" entry form](#add-a-new-item-entry-form)
+- [Allow users to archive records](#allow-users-to-archive-records)
+- [Add reporting functionality](#add-reporting-functionality)
+
+## Build and Test a Basic Web API Service App (Backend)
+
+### Create a Web API project using the XAF Solution Wizard
+
+1. Start the wizard and select a Web API only project.
+  ![](../../images/MAUI/SolutionWizardWebAPI.png)
+2. Choose Entity Framework as your ORM.
+  ![](../../images/MAUI/SolutionWizardEFCore.png)
+3. Choose Standard Authentication to generate the [JWT](https://en.wikipedia.org/wiki/JSON_Web_Token)  authentication scaffolding code.
+   ![](../../images/MAUI/SolutionWizardAuthStd.png)
+4. If you are using the [Universal Subscription](https://www.devexpress.com/subscriptions/universal.xml), the next page will allow you to select additional modules to add to your Web API service. Choose all available modules and click **Finish**.
```

Comment:
Not sure why we instruct people to add all available modules. From what I see, the Reports module is required for this tutorial. Others are optional (we may tell people that they may use them if they plan to extend their project.)

```suggestion
4. If you own a license to the DevExpress [Universal Subscription](https://www.devexpress.com/subscriptions/universal.xml), the next page allows you to select additional modules to add to your Web API Service. Make sure to select **Reports** as this module is required to complete the tutorial (last step). You can choose other modules if you plan to extend the application. Click **Finish**.
```